internal/controller/controllers/infraenv_controller.go

```diff
@@ -373,10 +373,31 @@ func (r *InfraEnvReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return reply
 	}
 
+	mapClusterDeploymentToInfraEnv := func(a client.Object) []reconcile.Request {
```

Comment:
What `a` stand for? Shouldn't it be `clusterDeployment`?

Reply:
Changed